core/templates/pages/moderator-page/moderator-page.controller.ts

```diff
@@ -47,11 +47,12 @@ angular.module('oppia').directive('moderatorPage', [
           };
 
           ctrl.getActivityCreateUrl = function(reference) {
-            return (
-              (reference.type === (
-                'exploration' ? '/create' : '/create_collection')) +
-              '/' + reference.id);
+            var path = (reference.type === 'exploration' ?
```

Comment:
Break after (

This is what we do normally...

Reply:
Done.